### DISCUSSION TITLE
最新passwall没有了SSR，用什么软件替代？

### DISCUSSION THREAD

**Discussion by liyiranlab** [immortalwrt/immortalwrt#1364] [Q&A]
最新主线passwall没有了SSR，用什么软件替代？

**Comment by liyiranlab** [comment]
手动安装 shadowsocksr-libev-ssr-local  2.5.6-r12 shadowsocksr-libev-ssr-redir 2.5.6-r12 两个包又可以用了。

**Reply by liyiranlab** [reply]
新版直接连安装包也没有了

**Reply by ATang007ZH** [reply]
最新版源码在哪里？
